src/main/java/frc/robot/maps/TempestMap.java

```diff
@@ -1,32 +1,121 @@
 package frc.robot.maps;
 
+import java.util.function.BooleanSupplier;
+
 import com.chopshop166.chopshoplib.RobotMapFor;
 import com.chopshop166.chopshoplib.maps.DifferentialDriveMap;
 import com.chopshop166.chopshoplib.outputs.EncodedSpeedController;
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.AnalogTrigger;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.Victor;
+
 @RobotMapFor("Tempest")
 public class TempestMap extends RobotMap {
 
     @Override
     public DifferentialDriveMap getDriveMap() {
         return new DifferentialDriveMap() {
+            @Override
+            public EncodedSpeedController getLeft() {
```

Comment:
It looks like these are mostly formatting changes, but they probably shouldn't be here either.